go/cmd/gvel/layers/logic/mint_credit_by_credit.go

```diff
@@ -0,0 +1,60 @@
+package logic
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/velo-protocol/DRSv2/go/cmd/gvel/constants"
+	"github.com/velo-protocol/DRSv2/go/cmd/gvel/entity"
+	"github.com/velo-protocol/DRSv2/go/cmd/gvel/utils/crypto"
+	"github.com/velo-protocol/DRSv2/go/libs/vclient"
+)
+
+func (lo *logic) MintCreditByCredit(input *entity.MintCreditByCreditInput) (*entity.MintCreditByCreditOutput, error) {
+	// 1. Get default account
+	defaultAccount := lo.AppConfig.GetDefaultAccount()
+
+	// 2. Prepare private key
+	accountBytes, err := lo.DB.Get([]byte(defaultAccount))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get account from db")
```

Comment:
change errors message failed to get account from db -> default account is not found